src/main/java/com/open/drive/user/UserController.java

```diff
@@ -0,0 +1,37 @@
+package com.open.drive.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.websocket.server.PathParam;
+import java.util.List;
+
+@RestController
+public class UserController {
+
+    private final UserRepo repo;
+
+    @Autowired
+    public UserController(final UserRepo userRepo) {
+        repo = userRepo;
+    }
+
+    @GetMapping("/users")
+    public List<User> getUsers() {
+        return repo.findAll();
+    }
+
+    @GetMapping("/user")
+    public ResponseEntity getUser(@PathParam("id") final String id) {
```

Comment:
`getUser` returns `ResonseEntity` ? Not `User` ?
Where we can retrieve the `User` with inputted `id` (if found)?

Reply:
ResponseEntity is some kind of packaging class where we have field body which is transformed to json with our user. It has also some other fields like HttpStatus etc